frontend/src/views/ApprovedPage/ApprovedPage.js

```diff
@@ -4,10 +4,10 @@ import {
   StyledText
 } from './StyledApprovedPage';
 
-const ApprovedPage = () => {
+const ApprovedPage = (props) => {
   return (
     <div>
-      <BackButton to="anvandare" />
+      <BackButton back={props} />
```

Comment:
denna ser ska tvärr se ut såhär för att kunna funka "props={props.props.props} to="/anvandare""